opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -674,3 +693,18 @@ def shutdown(self):
         if self._atexit_handler is not None:
             atexit.unregister(self._atexit_handler)
             self._atexit_handler = None
+
+    def force_flush(self, timeout_millis: int = None) -> bool:
+        """Requests the active span processor to process all spans that have not
+        yet been processed.
+
+        Args:
+            timeout_millis: The maximum amount of time to wait for spans to be
+                processed.
+
+        Returns:
+            False if the timeout is exceeded, True otherwise.
+        """
+        if timeout_millis is None:
```

Comment:
Is this `if` required? can't we call `force_flush(timeout_millis)` in both cases? I mean technically there can be a difference if a function forgot to specify a default argument or has a different one, but should we support that?